browser/ui/views/frame/split_view/brave_contents_container_view.cc

```diff
@@ -44,19 +46,24 @@ BraveContentsContainerView::BraveContentsContainerView(
       browser->profile(), use_rounded_corners));
   reader_mode_toolbar_->SetDelegate(this);
 #endif
+
+  if (base::FeatureList::IsEnabled(features::kSideBySide)) {
+    mini_toolbar_ =
```

Comment:
Do we need to reset |mini_toolbar_| set from base class's constructor?

```suggestion
    RemoveChildViewT(mini_toolbar_);
    mini_toolbar_ =
```